src/main/resources/reference.conf

```diff
@@ -511,6 +511,41 @@ resourceTypes = {
     }
     reuseIds = true
   }
+  spend-profile = {
+    actionPatternObjects = {
+      "create" = {
+        description = "Can create profiles"
+      }
+      "update" = {
+        description = "Can update profiles"
+      }
+      "delete" = {
+        description = "Can delete profiles"
+      }
+      "list" = {
+        description = "Can read profiles"
+      }
+      "link" = {
+        description = "Can link profiles"
+      }
+      "share_policy::owner" = {
+        description = "Can grant and revoke a users' owner permission"
+      }
+    }
+    ownerRoleName = "owner"
+    roles = {
+      admin = {
```

Comment:
do we need admin in addition to owner? If a user has `share_policy::owner` they can add themselves to the owner policy

Reply:
We were debating this ourselves. How have you handled the case where there is no one in the owner role? For example, someone leaves the company and they are the sole owner.

We put the admin in with the ability to make a new owner. That is all it is for.

Reply:
We don't handle orphaned resources well at the moment but I don't see how an admin role fixes it. Some other user would still have to be granted the admin role... why not just grant the other user the owner role (owners can make other owners too)?

Reply:
Our idea was this: when TDR creates a resource, it grants admin role to a group - some managed group like JadeAdmins. The owner would not be able to change that grant; we would not give the owner share_policy:admin. The only action the admin would have is share_policy:owner to add a new owner. (Of course, in practice that is all permissions, but it would require auditable actions.)

What do you think?

Reply:
After talking with Dan, I've removed the "admin" role.